src/main/java/com/revature/model/Profile.java

```diff
@@ -15,9 +13,9 @@
 import javax.persistence.ManyToMany;
 import javax.persistence.OneToMany;
 import javax.persistence.Table;
-
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
 import com.fasterxml.jackson.annotation.JsonIgnore;
-
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
 /**

```

Comment:
Why was this file even touched in the first place? It seems like there are no useful changes being made.

Reply:
Changes are made as per suggestion